src/gsi/read_seviri.f90

```diff
@@ -529,7 +529,7 @@ subroutine read_seviri(mype,val_sev,ithin,rmesh,jsatid,&
   close(lnbufr)
 
   call combine_radobs(mype_sub,mype_root,npe_sub,mpi_comm_sub,&
-     nele,itxmax,nread,ndata,data_all,score_crit,nrec)
+     nele,itxmax,nread,nread,ndata,data_all,score_crit,nrec)
```

Comment:
Why doesn't `read_seviri.f90` have `number_profiles` like other radiance read routines?

Reply:
@RussTreadon-NOAA thanks for catching this!  I added the counts statement and changed the variable name to number_profiles in read_seviri.f90 is now consistent with the other satellite instrument reads.